src/main/scala/it/pagopa/interop/backendforfrontend/common/system/ApplicationConfiguration.scala

```diff
@@ -57,6 +57,9 @@ object ApplicationConfiguration {
   val selfcareV2URL: String    = config.getString("backend-for-frontend.services.selfcare-v2")
   val selfcareV2ApiKey: String = config.getString("backend-for-frontend.api-keys.selfcare-v2")
 
+  val attributeRegistryProcessURL: String =
+    config.getString("backend-for-frontend.services.attribute-registry-process")
```

Comment:
this need to be added in `application.conf` and `application-standalone.conf`

Reply:
`application-standalone.conf` file is missing! Do I have to create it?

Reply:
you're right, in this service is not strictly required